project.yml

```diff
@@ -0,0 +1,143 @@
+name: Kiwix
+options:
+  xcodeVersion: 13.0
+  deploymentTarget:
+      iOS: 15.0
+      macOS: 12.0
+  generateEmptyDirectories: true
+  useTabs: false
+  groupSortPosition: top
+
+settings:
+  base:
+    DEVELOPMENT_TEAM: L7HWM3SP3L
+    GENERATE_INFOPLIST_FILE: YES
+    CURRENT_PROJECT_VERSION: 120
+    ENABLE_USER_SCRIPT_SANDBOXING: NO
+    ENABLE_MODULE_VERIFIER: YES
+    MODULE_VERIFIER_SUPPORTED_LANGUAGES: "objective-c objective-c++"
+    ASSETCATALOG_COMPILER_GENERATE_SWIFT_ASSET_SYMBOL_EXTENSIONS: YES
+    CLANG_ANALYZER_LOCALIZABILITY_NONLOCALIZED: YES
+    ENABLE_PREVIEWS: YES
+    INFOPLIST_FILE: Support/Info.plist
+    INFOPLIST_KEY_LSApplicationCategoryType: public.app-category.reference
+    INFOPLIST_KEY_UISupportedInterfaceOrientations_iPhone: UIInterfaceOrientationPortrait UIInterfaceOrientationLandscapeLeft UIInterfaceOrientationLandscapeRight
+    INFOPLIST_KEY_UISupportedInterfaceOrientations_iPad: UIInterfaceOrientationPortrait UIInterfaceOrientationPortraitUpsideDown UIInterfaceOrientationLandscapeLeft UIInterfaceOrientationLandscapeRight
+    INFOPLIST_KEY_NSPhotoLibraryUsageDescription: "Kiwix needs permission to saves images to your photos app."
+    INFOPLIST_KEY_LSSupportsOpeningDocumentsInPlace: YES
+    INFOPLIST_KEY_UIApplicationSupportsIndirectInputEvents: YES
+    SWIFT_OBJC_INTEROP_MODE: objcxx
+    SWIFT_OBJC_BRIDGING_HEADER: $(PROJECT_DIR)/Support/Kiwix-Bridging-Header.h
+    GCC_C_LANGUAGE_STANDARD: c11
+    CLANG_CXX_LANGUAGE_STANDARD: c++14
+    ENABLE_HARDENED_RUNTIME[sdk=macosx*]: YES
+    APP_STORE_ID: $(APP_STORE_ID)
+    INFOPLIST_KEY_UILaunchScreen_Generation: YES
+    INFOPLIST_KEY_UILaunchStoryboardName: LaunchKiwix.storyboard
+
+  configs:
+    debug:
+      DEAD_CODE_STRIPPING: YES
+      MODULE_VERIFIER_SUPPORTED_LANGUAGE_STANDARDS: "gnu11 gnu14++"
+    release:
+      DEAD_CODE_STRIPPING: YES
+      MODULE_VERIFIER_SUPPORTED_LANGUAGE_STANDARDS: "gnu11 gnu14++"
+      VALIDATE_PRODUCT: YES
+
+packages:
+  Fuzi:
+    url: https://github.com/cezheng/Fuzi
+    majorVersion: 3.0.0
+  Defaults:
+    url: https://github.com/sindresorhus/Defaults
+    majorVersion: 6.0.0
+  SwiftUIBackports:
+    url: https://github.com/shaps80/SwiftUIBackports.git
+    majorVersion: 1.0.0
+  OrderedCollections:
+    url: https://github.com/apple/swift-collections.git
+    majorVersion: 1.0.4
+
+targets:
+  Kiwix:
+    settings:
+      base:
+        MARKETING_VERSION: "3.3"
+        PRODUCT_BUNDLE_IDENTIFIER: self.Kiwix
+        INFOPLIST_KEY_CFBundleDisplayName: Kiwix
+    type: application
+    supportedDestinations: [iOS, macOS]
+    # info:
+      # path: Support/Info.plist
+      # CFBundleName: Kiwix
+    sources: 
+      - path: App
+      - path: Support/Kiwix-Bridging-Header.h
+        destinationFilters: [iOS]
```

Comment:
Why would this not be required for macOS?

Reply:
This had to be removed for macOS in the original project file as well, in order to make it compile, please see it here:
https://github.com/kiwix/apple/blob/37532b303f7b6fff80a2234745164e2cd1470b3b/Kiwix.xcodeproj/project.pbxproj#L106C23-L106C23
at the end of line 106 we have: "platformFilter = ios; };"

Reply:
OK